tests/tensorflow/common.libsonnet

```diff
@@ -45,4 +45,14 @@ local volumes = import 'templates/volumes.libsonnet';
 
     image: 'gcr.io/xl-ml-test/tensorflow-tpu-1x',
   },
+  ServingTest:: common.CloudAcceleratorTest {
+    local config = self,
+    gcsDir: error 'must set gcsDir.',
```

Comment:
Can you move these new top-level parameters (`gcsDir` through `model`) into an object that has the settings for a Serving test?  e.g. like `paramsOverride` in some of our model tests.

Reply:
Done